python/cuml/test/test_kneighbors_classifier.py

```diff
@@ -57,11 +57,13 @@ def test_neighborhood_predictions(nrows, ncols, n_neighbors,
     predictions = knn_cu.predict(X)
 
     if datatype == "dataframe":
-        assert isinstance(predictions, cudf.DataFrame)
+        assert isinstance(predictions, cudf.Series)
```

Comment:
Now that this is supporting different output types, we probably want to add these as parameters to the tests. I've found that relying on the `CumlArray` tests doesn't provide a good enough guarantee that individual estimators are supplying the proper output conversions